templates/rabbitmq.config.j2

```diff
@@ -1,10 +1,14 @@
 [
     {rabbit, [
+    {% if sensu_ssl_manage_certs %}
```

Comment:
The `sensu_ssl_manage_certs` was requested specifically by someone that wanted SSL to be used, but didn't want the cert files to be handled by this role. Please use a different variable or get rid of it altogether.